inspirehep/base/templates/header.html

```diff
@@ -55,13 +55,13 @@
         </span>
       </a>
       {% if request.endpoint in ['search.search', 'record.metadata'] %}
-        <div id="top-search-bar-small" class="col-sm-5 {{collection_name}}">
+        <div id="top-search-bar-small" class="col-sm-5 {% if collection %}{{collection.name}} {% endif %}">
```

Comment:
I think this fie might not be to be touched with the latest changes to `header.html`. The problems with `collection_name` should be solved now.